packages/components-js/tests/e2e/specs/default-dom.e2e.ts-snapshots/p-textarea.txt

```diff
@@ -0,0 +1,13 @@
+<div class="root">
+  <label class="label" id="label" for="textarea"> </label>
```

Comment:
Why do we need id="label"?

Reply:
<img width="343" alt="image" src="https://github.com/user-attachments/assets/2b57fc23-5d6e-40bb-80b6-12d7abdcd612">

For the htmlFor association

Reply:
Defined in our Label FC. Unused in this case but needed for aria-labelledby in other components